### PR TITLE
Quit Binding & Bindable Ctrl+C

### LINKED ISSUE #386: Can an exit key other than <ctr-c> be added?

### PR DESCRIPTION
**Pull request by Retropaint**
Closes #386 

Credit to @denisstrizhkin for #405. This is pretty much an improved version of it.

### CODE REVIEW

File: cmd/application.go [hunk 1]
```diff
@@ -16,9 +17,22 @@ func newApplication() *Application {
...
 
 	app.EnableMouse(cfg.Mouse)
+	app.SetInputCapture(onInputCapture)
 	return app
 }
 
+func onInputCapture(event *tcell.EventKey) *tcell.EventKey {
+ 	switch event.Name() {
+ 	case cfg.Keys.Quit:
+ 		app.Stop()
```
Comment (ayn2op) [inline cmd/application.go]: Just send `Ctrl+C` (using tcell.NewEventKey) here instead of calling `Stop`.

File: cmd/application.go [hunk 2]
```diff
@@ -16,9 +17,22 @@ func newApplication() *Application {
 	}
 
 	app.EnableMouse(cfg.Mouse)
+	app.SetInputCapture(onInputCapture)
 	return app
 }
 
+func onInputCapture(event *tcell.EventKey) *tcell.EventKey {
```
Comment (ayn2op) [inline cmd/application.go]: Also, convert this function into receiver function:
```go
func (app *Application) onInputCapture(event *tcell.EventKey) *tcell.EventKey
```

File: cmd/application.go (hunk 1 above)
Comment (Retropaint) [reply cmd/application.go]: I'm not following. Do you mean that `Ctrl+C` should be emulated with tcell.NewEventKey? If so, no clue how to do that.

**Review by ayn2op — APPROVED** [review]
LGTM! <3

**Pull request merged**